ddl/placement/utils.go

```diff
@@ -131,3 +133,32 @@ func BuildPlacementCopyBundle(oldBundle *Bundle, newID int64) *Bundle {
 	}
 	return newBundle
 }
+
+// GetLeaderDCByBundle return the leader's dc by Bundle if founded
+func GetLeaderDCByBundle(bundle *Bundle, dcLabelKey string) (string, bool) {
+	validRulesCount := 0
+	dc := ""
+	for _, rule := range bundle.Rules {
+		if isValidLeaderRule(rule, dcLabelKey) {
+			validRulesCount++
+			dc = rule.LabelConstraints[0].Values[0]
+		}
+	}
+	if validRulesCount != 1 {
+		if validRulesCount > 1 {
+			logutil.BgLogger().Warn("rule bundle have multi leader rules", zap.String("bundle-ID", bundle.ID))
```

Comment:
PD should have banned multi leader rules, doubt about its necessaries.

https://github.com/tikv/pd/blob/master/server/schedule/placement/rule_manager_test.go#L307-L325

Reply:
updated.